data_linter/validators/pandas_validator.py

```diff
@@ -362,100 +364,43 @@ def _parse_data_to_pandas(filepath: str, table_params: dict, metadata: dict):
-        if c["name"] not in metadata.get("partitions", [])
-    ]
-
-    # For string based file types convert make arrow readers read them in as strings
-    # validators will still treat these as dates but will run validation against strings
-    # cols expecting values to match a timestamp format
-    if "json" in metadata["file_format"] or "csv" in metadata["file_format"]:
-        md_obj = Metadata.from_dict(metadata)
-        cols = md_obj.columns
-
-        cols_to_force_str_read_in = []
-        for c in cols:
-            if c["type"].startswith("time") or c["type"].startswith("date"):
-                c["type"] = "string"
-                c["type_category"] = "string"
-                cols_to_force_str_read_in.append(c["name"])
-
-        md_obj.columns = cols
-        ac = ArrowConverter()
-        arrow_schema = ac.generate_from_meta(md_obj)
-
-        ts_as_str_schema = pa.schema([])
-        for cname in cols_to_force_str_read_in:
-            ts_as_str_schema = ts_as_str_schema.append(arrow_schema.field(cname))
-
     # Set the reader type
     if filepath.startswith("s3://"):
-        reader_fs = fs.S3FileSystem(region="eu-west-1")
-        fp_for_file_reader = filepath.replace("s3://", "", 1)
-
+        reader = wr.s3
     else:
-        reader_fs = fs.LocalFileSystem()
-        fp_for_file_reader = filepath
-
-    with reader_fs.open_input_stream(fp_for_file_reader) as f:
-        if "csv" in metadata["file_format"]:
-
-            # Safer CSV load for newlines_in_values set to True
-            if table_params.get("expect-header", True):
-                po = csv.ParseOptions(newlines_in_values=True)
-            else:
-                po = csv.ParseOptions(
-                    newlines_in_values=True, column_names=meta_col_names
-                )
-
-            if ts_as_str_schema:
-                co = csv.ConvertOptions(column_types=ts_as_str_schema)
-            else:
-                co = None
-
-            df = pa_read_csv_to_pandas(
-                input_file=f,
-                schema=arrow_schema,
-                expect_full_schema=False,
-                parse_options=po,
-                convert_options=co,
-            )
-            # dates/datetimes == string
-
-        elif "json" in metadata["file_format"]:
-
-            po = json.ParseOptions(
-                newlines_in_values=True,
-                explicit_schema=ts_as_str_schema if ts_as_str_schema else None,
-            )
-
-            df = pa_read_json_to_pandas(
-                input_file=f,
-                schema=arrow_schema,
-                expect_full_schema=False,
-                parse_options=po,
-            )
-            # dates/datetimes == string
+        reader = pd
+
+    # read the data
+    if "csv" in metadata["file_format"]:
+        df = reader.read_csv(filepath, dtype=str, low_memory=False)
+    elif "json" in metadata["file_format"]:
+        df = reader.read_json(filepath, dtype=str, lines=True)
```

Comment:
```suggestion
        df = reader.read_json(filepath, lines=True)
```

reason being is that we want to keep numbers and bools in json type if written like that (keeping the data as true to the origin file before we cast it)